tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1357,21 +1357,18 @@ def remove_static_routes(standby_tor, active_tor_loopback_ip):
     standby_tor.shell('ip route del {}/32'.format(active_tor_loopback_ip), module_ignore_errors=True)
 
 
-def increase_linkmgrd_probe_interval(duthosts, tbinfo):
+def recover_linkmgrd_probe_interval(duthosts, tbinfo):
     '''
     Increase the interval at which linkmgrd sends ICMP heartbeats to the server/PTF
     '''
     if 'dualtor' not in tbinfo['topo']['name']:
         return
 
-    probe_interval_ms = 1000
-
-    logger.info("Increase linkmgrd probe interval on {} to {}ms".format(duthosts, probe_interval_ms))
-
+    default_probe_interval_ms = 100
+    logger.info("Recover linkmgrd probe interval on {} to {}ms".format(duthosts, default_probe_interval_ms))
     cmds = []
     cmds.append('sonic-db-cli CONFIG_DB HSET "MUX_LINKMGR|LINK_PROBER" "interval_v4" "{}"'
-                .format(probe_interval_ms))
-    cmds.append("config save -y")
+                .format(default_probe_interval_ms))
     duthosts.shell_cmds(cmds=cmds)
```

Comment:
There is only a single command to run on each DUT. No need to use `shell_cmd` module. General `command` or `shell` module should be good enough.

Reply:
OK, I will change to `shell`